packaging/opensuse/snapd.spec

```diff
@@ -96,13 +109,13 @@ principles. Bundle your dependencies, run in a predictable environment, use
 moder kernel features for setting up the execution environment and security.
 The same binary snap package can be installed and used on many diverse systems
 such as Debian, Fedora and OpenSUSE as well as their multiple derivatives.
-.
+
 This package contains the official build, endorsed by snapd developers. It is
 updated as soon as new upstream releases are made and is designed to live in
 the system:snappy repository.
 
 %prep
-%setup -q -n %{name}-%{version}
+%setup -q
```

Comment:
Is `-n ...` default?

Reply:
`-n %{name}-%{version}` is default. You only set `-n` for non-default names.